### DISCUSSION TITLE
Are you aware of some solution in dotnet using F# to build k8s operators?

### DISCUSSION THREAD

**Discussion by 64J0** [dotnet/dotnet-operator-sdk#361] [Q&A] [answered]
First of all, congratulations on this project, it seems pretty cool and I'll take a look more deeply when getting some free time. Like I said in this discussion title, I want to start creating k8s operators using .NET, but instead of writing C# code, IMO it would be better to use F#. 

Until today I thought that it was possible to create operators only using Golang haha.

**Comment by buehler** [comment, marked as answer]
Hey @64J0 
Thank you :-) You absolutely can do that with this very package. I support both languages and I also have a template for F#. Since all code (under the hood) is compiled to intermediate language, it does not care if the lib is written in C#.

You may find introductions into F# operators here: https://github.com/buehler/dotnet-operator-sdk/tree/master/src/KubeOps.Templates/Templates/Operator.FSharp

**Reply by buehler** [reply]
About the last comment: yes. that is exactly the reason why I started this project ;-)

**Reply by 64J0** [reply]
All right, thank you for the reply and congratulations for the project.
